src/pages/en/community/guidelines.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Guidelines
+description: Quilt's community guidelines
+layout: /src/layouts/Page.astro
+---
+
+import Message from "@atoms/Message.astro";
+
+# What is this page?
+
+This page is a living document that outlines the community guidelines for Quilt commmunity. While those guidelines aren't authoritative,
+they set a standard for what we expect from our community members and staff.
+
+Authoritative policies can be found in our [RFCs](/about/faq#whats-an-rfc-how-do-they-work), along with our [Code of Conduct](/community/code-of-conduct) and [rules](/community/rules).
+If a guideline is in conflict with one of these, that document takes precedence over the guidelines.
+
+# Moderation
+
+<Message>
+	# Reporting a staff member
+
+	If you feel the need to report a member of staff, the right person to contact will depend on their roles:
+	- Developer: Contact ModMail or a Moderator directly
+	- Moderator: Contact a Manager or ModMail
+	- Manager: Contact a member of the Administrative Board or another Manager
+	- Administrative Board: Contact a Manager or another member of the Administrative Board
```

Comment:
I feel like this should probably also mention how to report a CT who isnt a mod?